source/modules/task-list.js

```diff
@@ -60,51 +91,106 @@
  */
 function createCustomTaskTag(taskName) {
     let taskContainer = document.createElement('li');
-    let taskButton = document.createElement('input');
     let taskLabel = document.createElement('label');
+    let taskButton = document.createElement('input');
     let editButton = document.createElement('button');
+    let doneButton = document.createElement('button');
 
-    // When user clicks on the task, it gets crossed off
-    taskButton.addEventListener('click', setFinished(taskContainer));
-    
     taskContainer.setAttribute('class', 'task');
     taskContainer.style.border = '3px solid black';
-    taskContainer.setAttribute('finished', 'false');
     
-    taskButton.setAttribute('type', 'button');
+    // When user clicks on the task, it gets crossed off
+    taskButton.setAttribute('type', 'radio');
+    taskButton.setAttribute('id', taskName);
+    taskButton.setAttribute('class', 'task-button');
+    taskButton.setAttribute('name', 'task-option');
+    taskButton.addEventListener('click', function(e) {
+        let currentTask = document.getElementById('current-task');
+            if (this.checked) {
+                currentTask.innerText = taskLabel.innerText;
+            }
+    });
 
-    taskButton.setAttribute('class', 'task-label');
-    taskButton.value = taskName;
-    taskLabel.contentEditable = false;
+    taskLabel.setAttribute('class', 'task-label');
+    taskLabel.setAttribute('for', taskName);
+    taskLabel.innerText = taskName;
 
     editButton.innerText = 'Edit';
+    doneButton.innerText = 'Done';
+    doneButton.style = "margin-left:50px";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse112-sp22-group5/cse112-sp22-group5/pullRequest?prid=9414373)
```suggestion
    doneButton.style = 'margin-left:50px';
```